src/metrikker/iatjenester.ts

```diff
@@ -10,24 +10,42 @@ interface IaTjenesteMetrikk {
     kilde: String;
     tjenesteMottakkelsesdato: String;
 }
+export enum IaTjenesteKilde {
+    SYKEFRAVÆRSSTATISTIKK = 'SYKEFRAVÆRSSTATISTIKK',
+    KALKULATOR = 'KALKULATOR',
+}
+const erMetrikkerSendtForKilde = (
+    orgnr: string | undefined,
+    sendteMetrikker: [TjenestePerOrgnr],
+    kilde: IaTjenesteKilde
+) => {
+    return sendteMetrikker.some(
+        (tjenestePerOrgnr) => tjenestePerOrgnr.orgnr === orgnr && tjenestePerOrgnr.kilde === kilde
+    );
+};
 
 export const erIaTjenesterMetrikkerSendtForBedrift = (
     orgnr: string | undefined,
-    sendteMetrikker: [string]
+    sendteMetrikker: [TjenestePerOrgnr],
+    kilde: IaTjenesteKilde = IaTjenesteKilde.SYKEFRAVÆRSSTATISTIKK
 ): boolean => {
     if (orgnr === undefined) {
         return true;
```

Comment:
Er det ikke litt rart å svare "ja" på spørsmålet "er ia-tjenester sendt for bedrift?" når vi ikke vet hvilken bedrift det er snakk om (altså når orgnr er undefined)? 😅

Uansett så kan du fjerne undefined fra denne funksjonen, fordi den blir kun brukt ett sted, og der sender vi inn orgnr ?? '' , som i praksis aldri er undefined.